### PR TITLE
Port a test for tuple names on interface implementation and constraints to VB

### PR DESCRIPTION
**Pull request by VSadov**
Port a test for tuple names on interface implementation and constraints to VB

### CODE REVIEW

**Comment by VSadov** [comment]
test this please

**Comment by VSadov** [comment]
test this please

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - please take a look. Test-only change.

File: src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb [hunk 1]
```diff
@@ -17423,6 +17423,222 @@ implicit operator AA
...
+                Return Current
+            End Get
+        End Property
+
+        Public Sub Dispose() Implements IDisposable.Dispose
+        End Sub
+
+        Public Function MoveNext() As Boolean
+            index += 1
+            Return index &lt; _backing.Length
```
Comment (jcouv) [inline src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb]: Consider using CDATA around the whole inlined code

File: src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb [hunk 2]
```diff
@@ -17423,6 +17423,222 @@ implicit operator AA
...
+                                                        Assert.True(constraint.IsGenericType)
+                                                        Assert.Equal(1, constraint.TypeArguments.Length)
+                                                        Dim typeArg As TypeSymbol = constraint.TypeArguments(0)
+                                                        Assert.True(typeArg.IsTupleType)
+                                                        Assert.Equal(2, typeArg.TupleElementTypes.Length)
+                                                        Assert.All(typeArg.TupleElementTypes,
+                                                                        Sub(t) Assert.Equal(SpecialType.System_Int32, t.SpecialType))
+                                                        Assert.False(typeArg.TupleElementNames.IsDefault)
+                                                        Assert.Equal(2, typeArg.TupleElementNames.Length)
+                                                        Assert.Equal({"key", "val"}, typeArg.TupleElementNames)
```
Comment (jcouv) [inline src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb]: Consider removing the two checks (is default and length) above, as they are redundant.

**Comment by cston** [comment]
LGTM

**Pull request merged**